Add v1 latex source

imkharn opened the pull request:
This adds latex that generates the V1 whitepaper. The original latex was lost and reconstructed.

This commit also includes an updated build_latex.yml with a more recent latex compiler. This reconstructed latex generates an almost identical PDF. Notably, every page contains the exact same words and every line ends with the same character. The only differences exist due to unavoidable compiler changes and are as follows:
Margins are increased slightly.
Spacing between lines is reduced slightly to compensate.
A couple footnotes that used to overflow into the right column are now fully contained in the left column.

imkharn committed:
Add V1 Latex

This adds latex that generates the V1 whitepaper. The original latex was lost and reconstructed.

This commit also includes an updated build_latex.yml with a more recent latex compiler. This reconstructed latex generates an almost identical PDF. Notably, every page contains the exact same words and every line ends with the same character. The only differences exist due to unavoidable compiler changes and are as follows:
Margins are increased slightly.
Spacing between lines is reduced slightly to compensate.
A couple footnotes that used to overflow into the right column are now fully contained in the left column.

MicahZoltu commented on .github/workflows/build_latex.yml:
```diff
@@ -8,44 +8,49 @@ jobs:
     steps:
     - name: Checkout code
-      uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e
+      uses: actions/checkout@v4
```
Is there a reason for changing the version of these?  If not we should leave them to their original pins.  If there is a reason to update, then we should use a commit hash rather than version tag.

MicahZoltu commented on .github/workflows/build_latex.yml:
```diff
@@ -8,44 +8,49 @@ jobs:
       with:
         root_file: main.tex
-        working_directory: v2/english
+        working_directory: v1/english
```
I believe we want to continue to publish v2 paper, not replace v2 paper publishing with v1 paper publishing.  To achieve this I'm guessing we'll need to add additional sections or something.

MicahZoltu reviewed (dismissed):
I didn't realize we already have latex building as part of CI.  I may submit a PR to make it build on every push, rather than build only on push to main and cut releases every time.

Note: You don't have to create a new PR to apply changes.  You can just edit the existing PR, even from within GitHub UI (switch to your PR fork/branch, edit files, commit).

MicahZoltu commented:
https://github.com/AugurProject/whitepaper/pull/90 will make it so PDF is built on every push to this repository, which will let us compare the output of this to the existing v1 PDF prior to merging.  We should get that PR merged first, and then build this one on top of it.

MicahZoltu commented:
That PR is merged.  This PR will need to have conflicts resolved, or just redo the changes in `build_latex.yml` with the feedback I provided.  Then we can compare the output PDF with the one committed to this repository.

MicahZoltu commented on .github/workflows/build_latex.yml:
```diff
@@ -8,44 +8,49 @@ jobs:
-        asset_name: augur-whitepaper-v2.pdf
+        asset_path: v1/english/whitepaper.pdf
+        asset_name: whitepaper.pdf
         asset_content_type: application/pdf
```
```suggestion
name: Build LaTeX document
on: [push]
jobs:
  build_latex:
    runs-on: ubuntu-latest
    steps:
    - name: Checkout code
      uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e
    - name: Build v1 document
      uses: xu-cheng/latex-action@2031f1a94940bfe35904fc1782bf18a9bdafc127
      with:
        root_file: main.tex
        working_directory: v1/english
        pre_compile: |
          rm -f augur-whitepaper-v1.pdf
        post_compile: |
          mv main.pdf augur-whitepaper-v1.pdf
    - name: Check v2 file
      run: |
        file ./v1/english/augur-whitepaper-v1.pdf | grep -q ' PDF '
    - name: Upload v1 file
      uses: actions/upload-artifact@4cec3d8aa04e39d1a68397de0c4cd6fb9dce8ec1 # v4.6.1
      with:
        name: augur-whitepaper-v1.pdf
        if-no-files-found: error
        path: ./v1/english/augur-whitepaper-v1.pdf
    - name: Build v2 document
      uses: xu-cheng/latex-action@2031f1a94940bfe35904fc1782bf18a9bdafc127
      with:
        root_file: main.tex
        working_directory: v2/english
        pre_compile: |
          rm -f augur-whitepaper-v2.pdf
        post_compile: |
          mv main.pdf augur-whitepaper-v2.pdf
    - name: Check v2 file
      run: |
        file ./v2/english/augur-whitepaper-v2.pdf | grep -q ' PDF '
    - name: Upload v2 file
      uses: actions/upload-artifact@4cec3d8aa04e39d1a68397de0c4cd6fb9dce8ec1 # v4.6.1
      with:
        name: augur-whitepaper-v2.pdf
        if-no-files-found: error
        path: ./v2/english/augur-whitepaper-v2.pdf
    - name: Bump version and push tag
      if: ${{ github.ref == 'refs/heads/master' }}
      id: bump_version
      uses: anothrNick/github-tag-action@3840ec22ac98e14d981375e3ae2d8d0392964521
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        WITH_V: true
        DEFAULT_BUMP: patch
        INITIAL_VERSION: 2.0.0
    - name: Create Release
      if: ${{ github.ref == 'refs/heads/master' }}
      id: create_release
      uses: actions/create-release@0cb9c9b65d5d1901c1f53e5e66eaf4afd303e70e
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        tag_name: ${{ steps.bump_version.outputs.new_tag }}
        release_name: ${{ steps.bump_version.outputs.new_tag }}
        draft: false
        prerelease: false
    - name: Upload v1 Release Asset
      if: ${{ github.ref == 'refs/heads/master' }}
      id: upload-release-asset 
      uses: actions/upload-release-asset@e8f9f06c4b078e705bd2ea027f0926603fc9b4d5
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        upload_url: ${{ steps.create_release.outputs.upload_url }}
        asset_path: ./v1/english/augur-whitepaper-v1.pdf
        asset_name: augur-whitepaper-v1.pdf
        asset_content_type: application/pdf
    - name: Upload v2 Release Asset
      if: ${{ github.ref == 'refs/heads/master' }}
      id: upload-release-asset 
      uses: actions/upload-release-asset@e8f9f06c4b078e705bd2ea027f0926603fc9b4d5
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        upload_url: ${{ steps.create_release.outputs.upload_url }}
        asset_path: ./v2/english/augur-whitepaper-v2.pdf
        asset_name: augur-whitepaper-v2.pdf
        asset_content_type: application/pdf
```

MicahZoltu commented:
I updated this branch to be based of recent changes to master, which got the PDF automatically building by GitHub Actions.  Most of the changes are just minor whitespace things (not a problem), but there are a couple significant changes that should be addressed before we merge this:
1. All of the appendixes appear to be missing from the updated whitepaper.
2. Many references lost some information, for example (original first, updated second):
![{89F7FB34-7292-433C-9859-8F3CF4CB35A1}](https://github.com/user-attachments/assets/746789c5-e731-47a4-a8e0-a6894ba9c8d6)
![{37387D1A-533C-4D14-A092-F75F1D7283EC}](https://github.com/user-attachments/assets/ff0c0624-3f54-4b38-9153-90fdef2a6128)

[original-augur-whitepaper-v1.pdf](https://github.com/user-attachments/files/19067813/original-augur-whitepaper-v1.pdf)
[updated-augur-whitepaper-v1.pdf](https://github.com/user-attachments/files/19067814/updated-augur-whitepaper-v1.pdf)

imkharn committed:
Replaced whitepaper.pdf with reconstructed LaTeX files (main.tex, augur.bib, image 1.pdf, image 2.pdf)

imkharn committed:
Merged revised into the already active original add latex

imkharn commented:
Updated this (#89) instead of starting new with the now closed #92 per micahs request.

MicahZoltu commented on v1/english/main.tex:
The original paper used `=` here, but this uses `≡`.  Was this an intentional change or accidental?
```suggestion
Let $d$ denote the amount of stake placed\linebreak on $\hat{\omega}_1$ during the initial report. Because the tentative\linebreak outcome for each round is known in this situation, we can\linebreak simplify our notation for the dispute bond sizes. Define a\linebreak shorthand $B_n$ to denote the bond size required for round $n$, so that $B_1 = 2d$ and $B_n = B(\hat{\omega}_{n-1}, n)$ for all $n \geq 2$. This will make for easier reading and comprehension.
```

MicahZoltu commented on v1/english/main.tex:
```suggestion
\end{cases}\tag{E1}
```

MicahZoltu commented on v1/english/main.tex:
```suggestion
The total supply of REP in each child universe depends\linebreak on how much REP migrated to it, and when that migra-\linebreak tion occurred. Migrating REP during a fork, before it\linebreak is clear which child universe has achieved consensus, ex-\linebreak poses the user to a small (but non-zero) amount of risk\linebreak (see Section~\ref{section:Uncertain Fork Participation}), which may discourage participation\linebreak during the forking period of contentious forks. In order\linebreak to encourage participation during a fork, users must be\linebreak compensated for the risk.
```

MicahZoltu reviewed (changes requested):
I just compared the output PDF against the original and it is nearly identical!  There were 3 minor changes that I commented on, but I think this is just about ready.

Commit:
Apply suggestions from code review

I went ahead and applied these to expedite finishing this up.  Let me know if you think these changes are wrong and I can undo them.

MicahZoltu commented:
I went ahead and applied the changes myself to expedite moving this forward, let me know if you disagree with any of the 3 changes.

MicahZoltu commented on v1/english/main.tex:
Hmm, this still isn't rendering even after the change that I thought would fix it.  Maybe it is a problem with the syntax or something?  I know very little about LaTeX, so I'm not sure how to troubleshoot it further.
![{C47AB74A-A2FF-4203-BA2B-9DCB5785E76E}](https://github.com/user-attachments/assets/937a5369-b1d3-43fd-a0c1-11a2d916ae50)

MicahZoltu commented on v1/english/main.tex:
```suggestion
The total supply of REP in each child universe depends\linebreak on how much REP migrated to it, and when that migra-\linebreak tion occurred. Migrating REP during a fork, before it\linebreak is clear which child universe has achieved consensus, ex-\linebreak poses the user to a small (but non-zero) amount of risk\linebreak (see Section~\ref{section:uncertain_fork_participation}), which may discourage participation\linebreak during the forking period of contentious forks. In order\linebreak to encourage participation during a fork, users must be\linebreak compensated for the risk.
```

MicahZoltu commented on v1/english/main.tex:
```suggestion
\subsection{Uncertain Fork Participation}
\label{uncertain_fork_participation}
```

Commit:
Another attempt to fix section reference.

MicahZoltu commented on v1/english/main.tex:
```suggestion
The total supply of REP in each child universe depends\linebreak on how much REP migrated to it, and when that migra-\linebreak tion occurred. Migrating REP during a fork, before it\linebreak is clear which child universe has achieved consensus, ex-\linebreak poses the user to a small (but non-zero) amount of risk\linebreak (see Section~\ref{subsec:uncertain_fork_participation}), which may discourage participation\linebreak during the forking period of contentious forks. In order\linebreak to encourage participation during a fork, users must be\linebreak compensated for the risk.
```

MicahZoltu commented on v1/english/main.tex:
```suggestion
\label{subsec:uncertain_fork_participation}
```

Commit:
Yet another attempt to fix references.

MicahZoltu commented:
Finally got the `ref` thing fixed.  Going to merge this, please let me know if any of the 3 minor changes I made to make the output match the original are incorrect and we can open a PR to apply the changes you had.  I'm guessing they were just minor typos/mistakes.
